oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/TokenHandlerTest.java

```diff
@@ -222,9 +222,10 @@ void shouldReturn200ForSuccessfulTokenRequest(
         String authCode = new AuthorizationCode().toString();
         AuthenticationRequest authenticationRequest =
                 generateAuthRequest(JsonArrayHelper.jsonArrayOf(vectorValue));
-        VectorOfTrust vtr =
+        List<VectorOfTrust> vtr =
                 VectorOfTrust.parseFromAuthRequestAttribute(
                         authenticationRequest.getCustomParameter("vtr"));
+        VectorOfTrust vtrLowestCredentialTrustLevel = VectorOfTrust.orderVtrList(vtr).get(0);
```

Comment:
`orderVtrList` uses the assumption that lists contain either identity or non-identity vectors